### ISSUE TITLE
How to handle unicode strings & lowercase letters not able to detect in ascii field

### ISSUE THREAD

**Issue by dominicarrojado** [lookback/meteor-emoji#3]
I force the Emoji to turn into image but now what If user paste a 😀 which is a unicode?

**Comment by dominicarrojado** [comment]
Cannot detect small laters on ascii I tried to put ':p' in the ascii but its able to detect capitalize ":P" and some letters even capitalize not working also.

**Comment by dominicarrojado** [comment]
@johanbrook

**Comment by johanbrook** [comment]
This should perhaps be two issues?
- Anyway, I think we should go with capitalized ASCII emojis. Otherwise, `:d` would be a `:D`, which looks funny.
- It's been out of scope for this package to parse Unicode emojis. Happy to look at a PR for that though.

**Comment by dominicarrojado** [comment]
Thanks for the reply man! @johanbrook Yea probably right about sticking to capitalized I'm just used to entering `:p`. But the issue tho some ascii's not working. Like you defined this ascii :') but its not able to find the emoji and also trying other letters doesn't work also like capital L for example.

Ok not a big deal in the second one. Hope somebody PR haha.

**Comment by dominicarrojado** [comment]
@johanbrook btw loving the package. So easy to use and customize

**Comment by johanbrook** [comment]
Thank you!

Are you sure `:')` isn't parsed? The tests for it go through: https://github.com/lookback/meteor-emoji/blob/master/spec/emojis-spec.coffee#L63-L71

**Comment by dominicarrojado** [comment]
Yeah. `:*` not working also. Btw I forced all to use images. I tried both {{#emoji}}{{/emoji}} and the javascript Emojis.parse(text). Still doesn't parse. Later I'll try not forcing to use images if it still doesn't parse.

**Comment by dominicarrojado** [comment]
OK I figured why `:')` is not being parsed cause I escaped it with an HTML escape script. But `:*` still doesn't work after removing all my parsers. Just the plain message.
